trainee_template/react_series.amlg

```diff
@@ -160,13 +160,17 @@
 						"action_features" output_features
 						"series" (map (lambda (get (current_value) "series")) series_output)
 					)
-					(if output_influential_cases
-						(assoc "influential_cases" (map (lambda (get (current_value) "influential_cases")) series_output))
+					(if output_details
+						(map
+							(lambda
+								(map (lambda (get (current_value) (current_index 1))) series_output)
```

Comment:
This map seems wrong.  Since `(zip requested_details)` results in an assoc of value->nulls,  doing a (get (current_value) ...) means you are always getting from a null?  Also missing comments because of complexity

Reply:
Yeah I'll add some comments. That `(current_value)` iterates through `series_output`.

Reply:
I added comments. Please let me know if it still doesn't seem clear enough.

Reply:
can you please put the inner map inside a let statement where you store (current_index 1) as  `detail` and replace the instance of `(current_index 1)` with `detail` so it's more readable? it's a little less efficient but much more readable and maintainable

Reply:
fine with me. Will do.